packages/gatsby-source-wordpress/README.md

```diff
@@ -140,10 +140,10 @@ module.exports = {
         // URLs for thumbnails, or any other media detail.
         // Defaults to false
         keepMediaSizes: false,
-        // use a custom normalizer which is applied after the built-in ones.
-        normalizer: function({ entities }) {
-          return entities
-        },
+        // add/remove wordpress data normalizers
+        // each normalizer is an object with name and normalizer property being a function that accepts
+        // object with entities array and different helpers and configuration settings
+        normalizers: normalizers => [dropUnusedMediaNormalizer, ...normalizers],
```

Comment:
Can we try not to use `dropUnusedMediaNormalizer` here (as it's not explained what it is at this point).

Wondering if this could be something like this:
```suggestion
        normalizers: normalizers => [
          ...normalizers,
          {
            name: "nameOfTheFunction",
            normalizer: function({ entities }) {
             // manipulate entities here
             return entities
           },
          }
        ],
```
to vizualize the structure of normalizer here. The `dropUnusedMediaNormalizer` is used as example below, so this won't be missed